packages/frontend/src/components/common/Filters/FilterRuleForm.tsx

```diff
@@ -152,9 +167,21 @@ const FilterRuleForm: FC<Props> = ({
                             <Menu.Item onClick={onConvertToGroup}>
                                 Convert to group
                             </Menu.Item>
-                            <Menu.Item color="red" onClick={onDelete}>
-                                Remove
-                            </Menu.Item>
+                            <Tooltip
```

Comment:
same for this tooltip: 
- variant
- multiline
- does it need a span

Reply:
done thanks (answered about the span above)